Makefile.env.example

```diff
@@ -0,0 +1,4 @@
+export STORAGE_BOS_INTEGRATION_TEST=on
+export STORAGE_BOS_CREDENTIAL=hmac:access_key:secret_key
+export STORAGE_BOS_NAME=bucketname
+export STORAGE_BOS_ENDPOINT=location
```

Comment:
Why `endpoint` is `location` here? Maybe it should be `https:bos-region-name.bcebos.com`.

Reply:
This is because I didn't change it here after splitting the PR, and I'm going to submit the change on the next PR.